README.md

```diff
@@ -192,21 +194,21 @@ real DAC. Should probably run on any other board supported by [STM32duino](https
 - Note that AUDIO_INPUT and mozziAnalogRead() return values in the STM32's full ADC resolution of 0-4095 rather than AVR's 0-1023.
 - twi_nonblock is not ported
 
-### Teensy 3.0/3.1
+### Teensy 3.1/3.2/3.4/3.5/LC
 
 Extra libraries required for use withTeensy 3.0/3.1:
-
+These are included in the standard Teensyduino install
```

Comment:
Does that mean Teensy users will not have to worry about installing these, today? Then I'd suggest removing the notice. Or is that only for a subset of Teensy's, or necessary in some circumstances?

Reply:
The teensyduino installer lets you choose which libraries to install, so conceivably somebody might uncheck the box for these libraries and not know how to proceed. This library dependency issue is not handled well by Arduino at this point...